plaso/parsers/fseventsd.py

```diff
@@ -159,8 +153,23 @@ def ParseFileObject(self, parser_mediator, file_object, **kwargs):
     Raises:
       UnableToParseFile: when the header cannot be parsed.
     """
-    page_header = self._ParseDLSPageHeader(file_object)
+    page_header_map = self._GetDataTypeMap('dls_page_header')
+    file_offset = 0
+    file_size = file_object.get_size()
+    try:
+      page_header, header_size = self._ReadStructureFromFileObject(
```

Comment:
Maybe?
```
     page_header, file_offset = self._ReadStructureFromFileObject(
         file_object, 0, page_header_map)
```

Reply:
Changed this a little bit, but I find the suggestion here a little too implicit.